nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/main/java/org/apache/nifi/web/security/jwt/JwtAuthenticationFilter.java

```diff
@@ -43,15 +46,27 @@ public Authentication attemptAuthentication(final HttpServletRequest request) {
         // TODO: Refactor request header extraction logic to shared utility as it is duplicated in AccessResource
 
         // get the principal out of the user token
-        final String authorization = request.getHeader(AUTHORIZATION);
+        final String authentication = request.getHeader(AUTHORIZATION);
```

Comment:
I changed this because "Authorization" as a header name for containing authentication credentials is misleading (a historically bad HTTP header name). At the first opportunity in code I thought if I changed this instead to use a variable named authentication, the rest of the code can be read as 'authentication'. I can change this back though.

Reply:
I agree the HTTP header name is misleading, but since that header is called "Authorization", I think we should be consistent. You could name it `authorizationHeaderContainingAuthenticationCredentials` to be explicit.

Reply:
I have named it authorizationHeader :)